crates/swc_ecma_transforms_base/src/rename/analyzer/mod.rs

```diff
@@ -378,28 +372,6 @@ impl Visit for Analyzer {
         self.add_decl(n.local.to_id(), true);
     }
 
-    fn visit_member_expr(&mut self, e: &MemberExpr) {
-        e.obj.visit_with(self);
-
-        if let MemberProp::Computed(c) = &e.prop {
-            c.visit_with(self);
-        }
-    }
-
-    fn visit_method_prop(&mut self, f: &MethodProp) {
-        f.key.visit_with(self);
-
-        f.function.visit_with(self)
-    }
-
-    fn visit_named_export(&mut self, n: &NamedExport) {
-        if n.src.is_some() {
-            return;
-        }
-
-        n.visit_children_with(self);
-    }
-
```

Comment:
The goal of removing this visitor is to merge the `analyzer` and `collector` lately.